rpmio/rpmglob.c

```diff
@@ -85,34 +84,27 @@ static int __glob_pattern_p(const char *pattern, int quote)
 
 /* librpmio exported interfaces */
 
-int rpmGlob(const char * patterns, int * argcPtr, ARGV_t * argvPtr)
+int rpmGlob(const char * pattern, int * argcPtr, ARGV_t * argvPtr)
```

Comment:
This change needs to be reflected in the corresponding header too. This being a public API function, it's technically silent breakage for anybody relying on the old behavior so generally it'd be better to introduce a new function to force the break, BUT. I don't think anybody is using this API outside rpm itself, and the fact that it accepts multiple patterns was never documented anywhere besides the "s" in the argument name. So we can always just call that a typo :laughing:

Just update the header and I think it'll be fine. Accepting multiple whitespace separated patterns there is just a terrible API. If we needed multiple patterns we could pass them in via ARGV_t, but not like this. It's a good thing to address the issue at the source, really.